amd_openvx/openvx/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@
 cmake_minimum_required(VERSION 3.0)
 project(openvx)
 
+# Default libdir to "lib", this skips GNUInstallDirs from trying to take a guess if it's unset:
+set(CMAKE_INSTALL_LIBDIR "lib" CACHE STRING "Library install directory")
+#including GNUInstall Dir definitions.
+include( GNUInstallDirs )
```

Comment:
do we need this - `include( GNUInstallDirs )` in every file or can we just cache it in the main file and let it stream down to all?

Reply:
Removed all subdir level inclusions in latest version